scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/wson.yaml

```diff
@@ -338,3 +338,44 @@ WSON-12-1EP_4x4mm_P0.5mm_EP2.6x3mm:
   #suffix: '_PullBack'
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
+
+
+SON-8_1.5x2.0mm_P0.5mm:
+  device_type: 'SON'
+  library: Package_SON
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/tps62840.pdf'
+  ipc_class: 'qfn_pull_back' #'qfn' | 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  #body_size_x_min: 2.85
+  body_size_x:
+    nominal: 1.5
+    tolerance: 0.1
```

Comment:
Use same dimensioning style as the datasheet. In this case minimum and maximum. Same with y size.

```yaml
  body_size_x:
    minimal: 1.4
    maximal: 1.6
```

or
```yaml
body_size_x: 1.4 .. 1.6
```

Reply:
Done